src/web/content/static/css/letterpress.css

```diff
@@ -0,0 +1,280 @@
+@font-face {
```

Comment:
New css. I haven't modified the old file at all. The idea is that in the future we might be able to swap css files for different themes.